docs/dev_guides/index.md

```diff
@@ -9,11 +9,10 @@ We encourage contributions from all users. Whether it be adding a feature, fixin
 
 ## Contributing to Splink
 
-Thank you for your interest in contributing to Splink! If this is your first time working with Splink, check our [Contributors Guide](/CONTRIBUTING.html).
+Thank you for your interest in contributing to Splink! If this is your first time working with Splink, check our [Contributors Guide](./CONTRIBUTING.md).
 
 When making changes to Splink, there are a number of common operations that developers need to perform. The guides below lay out some of these common operations, and provides scripts to automate these processes. These include:
 
-* [Building a Virtual Environment](./changing_splink/building_env_locally.md) - to replicate the conditions when Splink is installed by users.
 * [Linting and Formatting](./changing_splink/lint_and_format.md) - to ensure consistent code style and to reformat code, where possible.
```

Comment:
I don't see a ton of utility in this list. "Linting and Formatting" and "Testing" are already linked in the contributor's guide section about contributing code (the only type of contribution to which they are relevant). Likewise with building the docs locally.

Releasing a new package version is nice to have documented, but calling it out for new contributors seems unnecessary -- they almost certainly wouldn't have the permissions to do this.

And contributing to the blog seems like perhaps it should just be its own section in the contributor's guide.

Reply:
Yeah agreed, in fact the whole page feels unnecessarily repetitious and could probably be dropped entirely.

We probably need to review it separately.  Just want to make sure the quickstart is as obvious as possible - feel free to do whatever you think's best!

Reply:
Sounds good and happy to help with that. I think in this PR I won't put the quickstart in the list since it is not a "common operation that developers need to perform."